src/complex-list-field/complex-list-field.component.html

```diff
@@ -64,7 +64,7 @@
         <!-- ADD-FIELD-FROM-SCHEMA, UP/DOWN and DELETE buttons for each row group -->
         <tr *ngIf="values.size > 0">
           <td class="button-holder">
-            <add-field-dropdown [fields]="keys[i]" (onFieldAdd)="onFieldAdd(i, $event)" [schema]="schema.items.properties"></add-field-dropdown>
+            <add-field-dropdown [fields]="keys[i]" (onFieldAdd)="onFieldAdd(i, $event)" [schema]="schema.items.properties">+</add-field-dropdown>
```

Comment:
the `+` that you see in the UI is actually this `+` since now we don't have it inside we pass it from here.

(see `<ng-content>` comment)